ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -620,3 +620,43 @@ def test_jax_numpy_einsum(
         _use_xeinsum=False,
         **kw
     )
+
+
+# amin
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_axis=-1,
+        max_axis=0,
+        min_num_dims=1,
+        force_int_axis=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.amin"
+    ),
+    keep_dims=st.booleans(),
+)
+def test_jax_numpy_amin(
```

Comment:
the `where` and `initial` parameters should be added to testing as well.
The test for numpy frontends [`sum`](https://github.com/unifyai/ivy/blob/e02ae8eb46a20bcda72688fb3ac08080a33562f5/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py#L37) might also be a useful reference here.

Reply:
thanks a lot Abdarhman I will work to fix this issues.